kikit/fab/common.py

```diff
@@ -203,11 +203,22 @@ def getCompensation(footprint):
              layerToSide(footprint.GetLayer()),
              footprintOrientation(footprint, getCompensation(footprint))) for footprint in footprints]
 
+class NaturalSorter:
+    #https://stackoverflow.com/questions/4836710/is-there-a-built-in-function-for-string-natural-sort
+    def __init__(self, selector=lambda x: x):
+        self.convert = lambda text: int(text) if text.isdigit() else text.lower()
+        self.selector = selector
+        self.key = lambda k: [self.convert(c) for c in re.split('([0-9]+)', self.selector(k))]
+
+    def sort(self, l):
+        return sorted(l, key=self.key)
```

Comment:
I think what you're suggesting is to have a `sort` and `sorted` equivalents, though for natural sort. But in this case the `key=` parameter is implied already and would require you to write both the `convert()` part and the `key()` part inside the `key=` parameter in case you want a different `selector`.

I argue this is a more elegant solution.